### DISCUSSION TITLE
KDE Connect

### DISCUSSION THREAD

**Discussion by flyg4t0** [JaKooLit/Hyprland-Dots#92] [General]
Hello and thank you for this project.  I'm trying to get KDE Connect to work, as a Hyprland noob, I'm must be missing something.  I got KDE Connect working while using Gnome in the same system. But when I login to Hyprland, KDE Connect doesn't seem to function.  I can't find any other KDE Connect devices.

**Comment by JaKooLit** [comment]
you may need to start additional services etc.....

sorry cant help you much as I dont use kde connect. I have a separate plasma for any plasma specific wanted

**Reply by flyg4t0** [reply]
Thanks for the quick response.  Extra services, that's what I figured.  I'll continue to explore.  Could you point me to your plasma specific content?

**Reply by JaKooLit** [reply]
I dont have.... I just keep my plasma on my system that I have since like 3 yrs ago... and from time I boot and update

**Reply by flyg4t0** [reply]
Got it.  I also have a separate plasma system (Kubuntu 23.10).  I wanted to test out Hyprland and possibly use it as my daily driver.  Maybe in the future there will be a way to get KDEConnect or GSConnect to work.  I'll continue to find a way. If I find a working solution, I'll report back.  Thanks again.

**Comment by SherLock707** [comment]
@flyg4t0 , I run KDEconnect in hyprland on Arch.

Just make sure you have this in Execs.conf so it runs when you boot up.
`exec-once =  /usr/lib/kdeconnectd &`

If you use a [firewall](https://wiki.archlinux.org/title/Firewall), you need to open UDP and TCP ports 1714 through 1764.

If your KDE connect theme isn't following system QT theme, just add a `kdeglobals` file in your `~./config`

**Reply by utkarsh-20-rajput** [reply]
> @flyg4t0 , I run KDEconnect in hyprland on Arch.
> 
> Just make sure you have this in Execs.conf so it runs when you boot up. `exec-once = /usr/lib/kdeconnectd &`
> 
> If you use a [firewall](https://wiki.archlinux.org/title/Firewall), you need to open UDP and TCP ports 1714 through 1764.
> 
> If your KDE connect theme isn't following system QT theme, just add a `kdeglobals` file in your `~./config`


What if I use GSConnect, Hyprland Ubuntu ?
no gnome extension is working

**Reply by SherLock707** [reply]
I never used GSConnect.
Check if this helps [https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Help#waiting-for-service](https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Help#waiting-for-service)

**Reply by cyberpaladin** [reply]
> @flyg4t0 , I run KDEconnect in hyprland on Arch.
> 
> Just make sure you have this in Execs.conf so it runs when you boot up. `exec-once = /usr/lib/kdeconnectd &`
> 
> If you use a [firewall](https://wiki.archlinux.org/title/Firewall), you need to open UDP and TCP ports 1714 through 1764.
> 
> If your KDE connect theme isn't following system QT theme, just add a `kdeglobals` file in your `~./config`

This is exactly what worked for me.

`sudo ufw allow 1714:1764/udp`
`sudo ufw allow 1714:1764/tcp`
`sudo ufw reload`

Worked instantly, even through vpn.

**Comment by gfhdhytghd** [comment]
I have a fix for remote input
https://github.com/gfhdhytghd/hypr-kdeconnect-fix
